yarn.lock

```diff
@@ -1177,6 +1177,21 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@eslint/eslintrc@^0.4.0":
```

Comment:
Changes in` yarn.lock` but not in `package.json`, have you added new packages?

Reply:
I didn't do that. Probably Yarn bundled it with typescript compiler or react package. That would explain why you didnt  have eslint enabled.